gio/src/auto/file_attribute_info_list.rs

```diff
@@ -0,0 +1,48 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::FileAttributeInfoFlags;
+use crate::FileAttributeType;
+use glib::translate::*;
+
+glib::wrapper! {
+    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct FileAttributeInfoList(Shared<ffi::GFileAttributeInfoList>);
+
+    match fn {
+        ref => |ptr| ffi::g_file_attribute_info_list_ref(ptr),
+        unref => |ptr| ffi::g_file_attribute_info_list_unref(ptr),
+        type_ => || ffi::g_file_attribute_info_list_get_type(),
+    }
+}
+
+impl FileAttributeInfoList {
+    #[doc(alias = "g_file_attribute_info_list_new")]
+    pub fn new() -> FileAttributeInfoList {
```

Comment:
```suggestion
    pub fn new() -> Self {
```

Reply:
That's autogenerated code :P

Reply:
Oups, i shouldn't have reviewed that from my phone :P